tests/unit/models/addon-test.js

```diff
@@ -193,7 +193,6 @@ describe('models/addon.js', function () {
 
         it('addon', function () {
           let app = {
-            importWhitelist: {},
```

Comment:
Unrelated, but came across this when scanning the codebase for the word `whitelist`.
`importWhitelist` is not used anymore, so might as well remove it.